.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -31,9 +42,10 @@ body:
       required: false
 
   - type: input
-    id: renovate-version-if-selfhosting
+    id: platform-version-if-selfhosting
     attributes:
-      label: If you're self-hosting Renovate, tell us what version of Renovate you run.
+      label: If you're self-hosting Renovate, tell us what version of the platform you run.
+      description: We do not support running Renovate on End of Support or End of Life platform versions.
```

Comment:
Maybe we need to expand this `description` with something like: "Feel free to open a bug for outdated platforms, but expect it to not get fixed", or something to that effect.

Or maybe we do accept bugs on outdated platforms, in case they're really bad?

Policy/wording is up to the Renovate code wizards. 😉

Reply:
Maybe this is not the right place for this type of stuff? We are adding potential friction for the 90%+ for which this is not a problem. I'd prefer to just gather the versions used and not add the warnings